EDI/Operation/07_PrepareWorkflow.md

```diff
@@ -0,0 +1,78 @@
+- The export definition is activated, see [Activate definition](01_ManageDefinitions.md#activate-definition).
+- Your workflow processes the business document (for example, the invoice) for which you have created the EDIFACT export definition. 
+
+
+#### Procedure
+
+*Workflow > Select workflow > Select version*
+
+![Workflow versions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditor.png "[Workflow editor]")
+
+1. Select a place with which the entity is provided that you want to export with the EDIFACT message.
+
+2. Add an *Execute PHP code* core action.
+
+    ![Add Execute PHP code](../../Assets/Screenshots/EDI/Operation/WorkflowExecutePHPCode.png "[Add Execute PHP code]")
+
+3. Enter a label in the *Label* field with which you can easily identify the purpose of this action, for example **Export to Customer 2**.
+
+4. Click the *PHP code* field in the *Configuration* section of the action settings.   
+    A pop-up window for editing the PHP code is displayed.
+
+    ![PHP code input window](../../Assets/Screenshots/EDI/Operation/WorkflowPHPInputWindow.png "[PHP code input window]")
+
+5. Insert the following lines of code (example).
+
+    ```
+    <?php
+ 
+    $deliveryNote = $in0;
+    $exportController = new \Actindo\Modules\Actindo\DataHubExporter\ExportController();
+    $exportRequest = new  \Actindo\Modules\Actindo\DataHubExporter\Request\ExportRequest(4);   
+    $exportRequest->connectionIds = []; 
+    $exportRequest->definitionId = ;    
+    $exportRequest->entityId = $in0->getId();
+    $exportRequest->entityClass = /Actindo\Core\Database\ClassUtils::getRealClass(get_class($in0));  
+    $exportController->export($exportRequest);
+    return [$in0];
+    ```
+    ![PHP code example](../../Assets/Screenshots/EDI/Operation/WorfflowPHPCodeExample.png "[PHP code example]")
+
+    **Explanation**
+    |php code | Meaning |
+    |--------|----|
+    |<?php|PHP's opening tag     |
+    |$deliveryNote = $in0;| Type of the entity that is to be exported (e.g. business document type that have been loaded at the start point and is input in *in0* input port), see the *Order Management* module for information on type <!---Stimmt das?-->|
+    |$exportController = new \Actindo\Modules\Actindo\DataHubExporter\ExportController();| Get exporter|
+    | $exportRequest = new \Actindo\Modules\Actindo\DataHubExporter\Request\ExportRequest(4);|Create new export request|  
+    |$exportRequest->connectionIds = [];| List of connection ID(s) (comma-separated)|
+    |$exportRequest->definitionId = []; | ID of the export definition, for example, $exportRequest->definitionId = 112;|
+    |$exportRequest->entityId = $in0->getId(); | Primary identifier of the entity that is to be exported (e.g. business document that have been loaded at the start point and is input in *in0* input port).|
```

Comment:
business document that has been loaded...

Reply:
done